### PR TITLE
close #483

### LINKED ISSUE #483: 训练一个分类器 | 报错解决方案
在[训练一个分类器](https://pytorch.apachecn.org/docs/0.3/blitz_cifar10_tutorial.html)这里，4. 训练网络第21行代码会报错，建议改为：running_loss += loss.item()

### PR DESCRIPTION
**Pull request by LovelyBuggies**

### CODE REVIEW

**Comment by LovelyBuggies** [comment]
@jiangzhonglian Why the CI check failed?

**Comment by LovelyBuggies** [comment]
I just realized that THIS IS WHAT 0.3 doc has done. No need changes.

**Comment by jiangzhonglian** [comment]
后续不再更新 1.4以前的版本，大家也注意哈～  看最新版本就行！
